.github/workflows/stale.yaml

```diff
@@ -0,0 +1,30 @@
+name: Label stale PRs and issues
+
+on:
+  schedule:
+    - cron: "30 1 * * *" # Runs daily at 1:30 AM UTC
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+      - uses: actions/stale@v9
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          days-before-issue-stale: 30
+          days-before-issue-close: -1 # never automatically close stale issues
+          stale-issue-label: "stale"
+          stale-issue-message: "This issue has been automatically marked as stale due to inactivity."
+          exempt-issue-labels: "stale"
+          # close-issue-message: 'Closing this issue due to prolonged inactivity.'
+          days-before-pr-stale: 30
+          days-before-pr-close: -1 # never automatically close stale PRs
+          stale-pr-label: "stale"
+          stale-pr-message: "This pull request has been automatically marked as stale due to inactivity."
+          # close-pr-message: 'Closing this pull request due to prolonged inactivity.'
+          exempt-pr-labels: "stale,help wanted to test"
```

Comment:
@davidfestal should the issues with "help wanted" on them be exempt from being marked stale? I'm thinking NO, but maybe you want them to persist more than 30 days without being addressed?